code/modules/spells/spell_types/touch/smite.dm

```diff
@@ -57,3 +57,14 @@
 	icon = 'icons/obj/weapons/hand.dmi'
 	icon_state = "disintegrate"
 	inhand_icon_state = "disintegrate"
+
+/obj/item/melee/touch_attack/smite/suicide_act(mob/living/user)
+	message_admins("[ADMIN_LOOKUPFLW(user)] suicided with [src] at [ADMIN_VERBOSEJMP(user)]")
+	user.log_message("suicided with [src].", LOG_ATTACK)
+	log_game("[key_name(user)] suicided with [src] at [AREACOORD(user)]")
```

Comment:
this is redundant because we will do this stuff in `suicide_log()`, and the message about the player dying of death will post to game and world logs all the same. i also don't see why admins need to be messaged over this tbh

Reply:
I figured it was because of it being a real explosion and not just a gib, so it could damage some stuff around it